src/index/sparse/sparse_inverted_index_config.h

```diff
@@ -58,26 +58,25 @@ class SparseInvertedIndexConfig : public BaseConfig {
          *    This will make the cached max score larger than the actual max
          *    score, so that it makes the filtering less aggressive, but
          *    guarantees the correctness.
-         * 2. In WAND searching process, we use the sum of the max scores to
-         *    filter the candidate vectors. If the sum is smaller than the
-         *    threshold, skip current vector. If approximate searching is
-         *    accepted, we can make the skipping more aggressive by downscaling
-         *    the max score with a ratio less than 1.0. Since the possibility
-         *    that the max score of all dims in the query appears on the same
-         *    vector is relatively small, it won't lead to a sharp decline in
-         *    the recall rate within a certain range.
+         * 2. For dimension maxscore based algorithms like WAND and MaxScore,
+         *    they use the sum of the max scores to filter the candidate
+         *    vectors. If the sum is smaller than the threshold, skip current
+         *    vector. If approximate searching is accepted, we can make the
+         *    skipping more aggressive by downscaling the max score with a
+         *    ratio less than 1.0. Since the possibility that the max score
+         *    of all dims in the query appears on the same vector is
+         *    relatively small, it won't lead to a sharp decline in the
+         *    recall rate within a certain range.
          */
-        KNOWHERE_CONFIG_DECLARE_FIELD(wand_bm25_max_score_ratio)
+        KNOWHERE_CONFIG_DECLARE_FIELD(dim_max_score_ratio)
             .set_range(0.5, 1.3)
             .set_default(1.05)
             .description("ratio to upscale/downscale the max score of each dimension")
```

Comment:
if set to below 1, make sure to check if you want to set refine_factor to a lower value